changelog.txt

```diff
@@ -5,6 +5,131 @@ Changes in version 4.26.0 (2024.02.07)
 
 ### Liquibase 4.26.0 is a major release
 
+> [!IMPORTANT]
+> Liquibase 4.26.0 contains several Notable Changes for Liquibase Pro users: Advanced IF conditionals, Simpler Regex-based pattern checks, and Checks Run Reports.
+
+> [!NOTE]
+> See the [Liquibase 4.26.0 Release Notes](https://docs.liquibase.com/start/release-notes/liquibase-4.26.0.html) for the complete set of release information.
+
+## Notable Changes
+
+### [PRO]
+#### Liquibase Flow: Advanced IF conditionals
+* In Liquibase flow files, use AND (&&), OR (||), and NOT (!) logic options, in single or grouped conditions, as well as substring matching and checking if a key or file exists.
+* Learn more https://docs.liquibase.com/flow
+
+#### Quality Checks: Simpler Regex-based pattern checks
+* Check four advanced patterns without having to figure out the complicated regex needed to do it—just supply the patterns and Quality Checks will link and execute them.
+* Learn more https://docs.liquibase.com/flow
+
+#### Observability: Checks Run Reports
+* Understand and share the data on the execution of Quality Checks with a new Checks Run Report. Easily scan and understand your results with the summary section, Analyze your checks by viewing details by ChangeSet, by check, and by database. And collaborate with your team with shareable report.
+* Learn more https://docs.liquibase.com/reports
```

Comment:
link is invalid, let's check with Mario the correct one

Reply:
https://docs.liquibase.com/observability